.circleci/config.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - checkout
       - <<: *install
       - run: npm run lint
+      - run: npx commitlint --from origin/master
```

Comment:
Wouldn't it make sense to make this a part of `npm run lint`?

Reply:
Unsure, it might have undesired effects (wrong branches etc as it's tied to git tree) and it's extremely slow:

```shell
$ time npx commitlint --from origin/master
...

        8.76 real         6.47 user         2.39 sys
```

That said, I personally don't use `npm run lint` command and instead only use lint plugins inside my editor. So I'll take whatever the consensus is for this command.

Reply:
I don't think there's a strong convention, it's up to every maintainer's preference.